UI/src/view/point_asset/trafficSignReadOnlyLayer.js

```diff
@@ -66,8 +66,8 @@
         me.removeLayerFeatures();
     };
 
-    eventbus.on(parentLayerName + ':hideReadOnlyTrafficSigns', hideReadOnlyTrafficSigns);
-    eventbus.on(parentLayerName + ':showReadOnlyTrafficSigns', showReadOnlyTrafficSigns);
+    eventbus.on(parentLayerName + '-readOnlyTrafficSigns:show', hideReadOnlyTrafficSigns);
+    eventbus.on(parentLayerName + '-readOnlyTrafficSigns:hide', showReadOnlyTrafficSigns);
```

Comment:
The events are switched, to hide the ReadOnly TrafficSigns we are using the event to show them and to show we are using the event to hide.